vars/pyenv.groovy

```diff
@@ -4,11 +4,11 @@ import com.ableton.VirtualEnv
 
 /**
  * Create a new {@link com.ableton.VirtualEnv} instance using pyenv.
- * @param python Python version, as given by pyenv versions --list.
+ * @param pythonVersion Python version, as given by pyenv versions --list.
  * @param pyenvRoot Path of the pyenv installation. If {@code null}, then the pyenv root
  *        will be detected from the environment in {@code PYENV_ROOT}.
  * @return New {@link com.ableton.VirtualEnv} instance.
  */
-VirtualEnv createVirtualEnv(String python, String pyenvRoot = null) {
-  return Pyenv.createVirtualEnv(this, python, pyenvRoot ?: env.PYENV_ROOT)
+VirtualEnv createVirtualEnv(String pythonVersion, String pyenvRoot = null) {
+  return new Pyenv(this, pythonVersion, pyenvRoot ?: env.PYENV_ROOT).createVirtualEnv()
```

Comment:
I don't mind creating a `Pyenv` object, but that object should be able to create any virtualenv, not just ones that use this specific `pythonVersion`. Said differently: `python`/`pythonVersion` should still be an argument of `createVirtualEnv`, like in `VirtualEnv.create`.

Reply:
`pyenvRoot` does make sense as a parameter to the constructor of `Pyenv`.

Reply:
True. Fixed in 7b6ad08.